src/network/networkd-wiphy.c

```diff
@@ -203,3 +238,36 @@ int manager_genl_process_nl80211_wiphy(sd_netlink *genl, sd_netlink_message *mes
 
         return 0;
 }
+
+int manager_udev_process_wiphy(Manager *m, sd_device *device) {
+        sd_device_action_t action;
+        const char *name;
+        Wiphy *w;
+        int r;
+
+        assert(m);
+        assert(device);
+
+        r = sd_device_get_action(device, &action);
+        if (r < 0)
+                return log_device_warning_errno(device, r, "Failed to get udev action, ignoring device: %m");
```

Comment:
Similar here, I would move the "ignoring device" up one level. But not sure how you'd choose the right log level one level up though